firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -540,6 +540,11 @@ protected FirebaseApp(Context applicationContext, String name, FirebaseOptions o
     publisher = componentRuntime.get(Publisher.class);
   }
 
+  @VisibleForTesting
+  static String getSharedPrefsName(String appName) {
```

Comment:
I would vote to make this private and hard-code in the test

Reply:
done, also did a smal refactor of the tests